cmd/snap-update-ns/change.go

```diff
@@ -56,10 +57,116 @@ func (c Change) String() string {
 var (
 	sysMount   = syscall.Mount
 	sysUnmount = syscall.Unmount
+	osLstat    = os.Lstat
+	// NOTE: we're not using os.MkdirAll as it is not careful about symlinks.
+	osMkdirAll = SecureMkdirAll
```

Comment:
why not call the variable `mkdirAll` then?

Reply:
I wanted to keep the same semantics as os.MkdirAll (to make the two interchangeable). I guess it's not a great reason, I can tweak this.